src/components/Frame/Frame.scss

```diff
@@ -180,7 +180,16 @@ $skip-vertical-offset: rem(10px);
   position: relative;
   padding-bottom: var(--global-ribbon-height, 0);
   flex: 1;
+  overflow: auto;
+
   @include layout-flex-fix;
+
+  .hasTopBar & {
+    height: calc(100vh - #{top-bar-height()});
```

Comment:
I think this breaks the admin in mobile safari

Reply:
Interesting, I tested in desktop safari. 
I haven't test with safari webview. 
Can you elaborate on the problem and do you have other suggestion for solving this issue.

We don't have to make this change to the Polaris frame content but we apply same logic in a wrapper element where we render the flow editor.

Reply:
This outlines it pretty well https://chanind.github.io/javascript/2019/09/28/avoid-100vh-on-mobile-web.html